packages/actor-query-operation-service/lib/ActorQueryOperationService.ts

```diff
@@ -37,7 +37,7 @@ export class ActorQueryOperationService extends ActorQueryOperationTypedMediated
       .delete(KeysRdfResolveQuadPattern.sources);
     const sourceType = this.forceSparqlEndpoint ? 'sparql' : undefined;
     subContext = subContext.set(KeysRdfResolveQuadPattern.sources, [{ type: sourceType, value: endpoint }]);
-
+    subContext = subContext.delete(KeysInitSparql.queryString);
```

Comment:
This operation can be chained onto the previous line, so we don't need the second override of `subContext`.